index.bs

```diff
@@ -1768,6 +1787,13 @@ a [=source type=] |sourceType|, and a [=duration=] |expiry|:
 1. If |map|["`event_report_windows`"] does not [=map/exist=], return the result
     of [=obtaining default effective windows=] given |sourceType|, |sourceTime|,
     and |expiry|.
+1. Return the result of [=parsing report windows=] with |map|,
+    |sourceTime|, and |expiry|.
+
+To <dfn>parse report windows</dfn> given a [=map=] |map|, a
+[=moment=] |sourceTime|, and a [=duration=] |expiry|:
+
+1. [=Assert=]: |map|["`event_report_windows`"] [=map/exists=].
```

Comment:
Maybe we can just pass the values in so as no assertion is needed?

Reply:
Wouldn't we need an assertion **somewhere**?

Reply:
IIUC this algorithm is only invoked after checking that the value exists.